_pages/en_US/installing-boot9strap-(mset9-cli).txt

```diff
@@ -34,9 +34,9 @@ In this section, you will prepare the SD card data necessary for the MSET9 explo
 
 1. Power on your console **with your SD card inserted**
 1. Open Mii Maker
-1. Wait for your console to reach the "Welcome to Mii Maker" screen, then exit Mii Maker
+1. Wait for your console to reach the ["Welcome to Mii Maker"](/images/screenshots/mset9/mii-welcome.png) screen, then exit Mii Maker
     + You may see [this screen](/images/screenshots/mset9/mii-extdata.png), which indicates the necessary data has been created
-    + If you just reach the normal Mii Maker screen, exit Mii Maker and continue to the next step
+    + If you just see [the normal Mii Maker menu](/images/screenshots/mset9/mii-existing.png), exit Mii Maker and continue to the next step
```

Comment:
style: we can remove the word 'just' here, I think

Reply:
Okay I lost this in the pile. Should this be a new PR at this point or can you merge parts of PRs?